W-16993325-pce4-documentation-kt (#344)
mulesoft/docs-private-cloud: modules/ROOT/pages/install-openshift.adoc

= Installing Anypoint Platform PCE on OpenShift

> Before installing Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) on OpenShift, you must configure Security Context Constraints (SCC) to grant the required permissions for pods and create roles for Prometheus, allowing it to collect metrics from the services. After these configurations are complete, proceed with installing Anypoint Platform PCE.

Reviewer: Suggestion: After GA, consider adding a Before You Begin header here.

[NOTE]

> You must have an operational OpenShift cluster prior installing Anypoint Platform PCE. If you do not have one set up, refer to the OpenShift documentation for setup instructions.

Reviewer: ```suggestion
You must have an operational OpenShift cluster prior to installing Anypoint Platform PCE. If you don’t have one set up, refer to the OpenShift documentation for setup instructions.
```

> You must have an operational OpenShift cluster prior installing Anypoint Platform PCE. If you do not have one set up, refer to the OpenShift documentation for setup instructions.

Reviewer: ```suggestion
You must have an operational OpenShift cluster prior to installing Anypoint Platform PCE. If you do not have one set up, refer to the OpenShift documentation for setup instructions.
```

> You must contact MuleSoft Professional Services to perform the installation.

Reviewer: For which installation: Openshift or PCE?
